hypothesisfuzzer/fuzz_server.py

```diff
@@ -95,9 +99,9 @@ def _load_config(self, config_path):
                         ConfigMissingOptionException("Configuration file" +
                                                      "missing a 'repos'" +
                                                      "attribute")
-                self.log.debug('File config.yml loaded.')
+                    self.log.debug('File config_path loaded.', exc_info=True)
```

Comment:
why is this part of the `if` block? Also should be info I think (since loading config is important)